src/mococrw/key.h

```diff
@@ -89,6 +91,39 @@ class AsymmetricPublicKey
      */
     static AsymmetricPublicKey readPublicKeyFromPEM(const std::string& pem);
 
+    /**
+     * @brief Returns a public key object based on the provided EC point
+     *
+     * This function creates a key object based on the compressed point in octet form
+     * and the elliptic curve group given by keySpec.
+     * The leading identifier (0x03 for compressed, 0x04 for uncompressed points. 0x07 for hybrid points) have to be in
+     * place.
+     *
+     * Remark: Ed-Curves (Ed448, Ed25519) are not supported. If you use one of these curves an MoCOCrWException is
+     * thrown.
+     * @throws MoCOCrWException if the given spec is not a elliptic curve spec or an Ed-spec
+     * @param keySpec The key specification for creating the key
+     * @param point The point(s) representing the public key
+     * @return An asymmetric public key
+     */
+    static AsymmetricPublicKey fromECPoint(const std::unique_ptr<ECCSpec> keySpec,
+                                           const std::vector<uint8_t> &point);
+
+    /**
+     * @brief This returns the uncompressed points in octet representation of the given asymmetric public key.
+     *
+     * Depending of the form the correct identifier will be prepended to the actual point (0x03 for compressed,
```

Comment:
Doesn't the sentence above indicate that this will always return the uncompressed version?

Reply:
Done.
We support compressed, uncompressed and hybrid points.